hw2/stackCalculator/IStack.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace StackCalculator
+{
+    internal interface IStack
+    {
+        /// <summary>
+        /// Добавляет элемент в стек
+        /// </summary>
+        /// <param name="element">Добавляемый элемент</param>
+        void Push(double element);
+        /// <summary>
```

Comment:
```suggestion
        void Push(double element);
        
        /// <summary>
```
Методы всегда разделяются пустой строкой